Add doc for making the first custom release (#6544)
opentripplanner/OpenTripPlanner: script/CUSTOM_RELEASE_README.md

enable it by changing the config.

## How To Make The First Release

> The release script looks for the _last release_ to resolve the next version. It uses the the version

Reviewer: ```suggestion
The release script looks for the _last release_ to resolve the next version. It uses the version
```

> in the Maven _pom.xml_, stip of `-SNAPSHOT` and append `<organization name>` and a sequence number

Reviewer: ```suggestion
in the Maven _pom.xml_, strips of `-SNAPSHOT` and appends `<organization name>` and a sequence number
```

Reviewer 2: strips off

(cannot make a suggested change to a suggested change...)

> (`N`). The release fails if there are no tags matching `vX.Y.Z-<organization name>-N`. There is two

Reviewer: ```suggestion
(`N`). The release fails if there are no tags matching `vX.Y.Z-<organization name>-N`. There are two
```
